src/Stan/Report/Html.hs

```diff
@@ -38,12 +49,55 @@ stanMain an = main_
       # divIdClassH "Analysis Info" "" ()
       )
     # divIdClassH "Graphs" "" (p_ "Maybe later")
-    # divIdClassH "Observations" "" (pre_ an)
+    # divIdClassH "Observations" "" (stanObservations an)
     # divIdClassH "Configurations" "" (p_ "Later")
     # divIdClassH "Summary" "" (p_ "Later")
     # divIdClassH "Inspections" "" (p_ "Later")
     )
 
+stanObservations Analysis{..} =
+    map stanPerFile
+    $ filter (not . null . fileInfoObservations)
+    $ Map.elems analysisFileMap
+
+stanPerFile FileInfo{..} = divIdClass "" ""
+    ( h3_ (toText fileInfoPath)
+    # p_ ("Module: " # maybe mempty (unModuleName . observationModuleName) (S.safeHead fileInfoObservations))
```

Comment:
I saw a similar code in terminal pretty-printing. Do you think it's worth it to store `ModuleName` in the `FileInfo`? Or at least create a helper function like:

```haskell
extractModuleName :: Observations -> Text
```